Documentation/Updating.md

```diff
@@ -67,8 +67,25 @@ If your project was created using the [Mixed Reality Toolkit NuGet packages](MRT
 
 ## Updating 2.3.0 to 2.4.0
 
+[Folder renames](#folder-renames-in-240)
 [API changes](#api-changes-in-240)
 
+### Folder renames in 2.4.0
+
+The MixedRealityToolkit folders have been renamed and moved into a common hierarchy in version 2.4. If an application
+uses hard coded paths to MRTK resources, they will need to be updated per the following table.
+
+| Previous Folder | New Folder |
+| --- | --- |
+| MixedRealityToolkit | MRTK\Core |
+| MixedRealityToolkit.Examples | MRTK\Examples |
+| MixedRealityToolkit.Extensions | MRTK\Extensions |
+| MixedRealityToolkit.Providers | MRTK\Providers |
+| MixedRealityToolkit.SDK | MRTK\SDK |
+| MixedRealityToolkit.Services | MRTK\Services |
+| MixedRealityToolkit.Tests | MRTK\Tests |
+| MixedRealityToolkit.Tools | MRTK\Tools |
+
```

Comment:
add important note of generated folder stays same

Reply:
done (and flipped the \ to /)